test/e2e/flask/btc/common-btc.ts

```diff
@@ -1,34 +1,36 @@
 import { Mockttp } from 'mockttp';
 import FixtureBuilder from '../../fixture-builder';
 import { withFixtures, unlockWallet } from '../../helpers';
-import { DEFAULT_BTC_ACCOUNT } from '../../constants';
+import { DEFAULT_BTC_ACCOUNT, DEFAULT_BTC_BALANCE } from '../../constants';
 import { MultichainNetworks } from '../../../../shared/constants/multichain/networks';
 import { Driver } from '../../webdriver/driver';
 import { createBtcAccount } from '../../accounts/common';
 
-const GENERATE_MOCK_BTC_BALANCE_CALL = (
-  address: string = DEFAULT_BTC_ACCOUNT,
-): { data: { [address: string]: number } } => {
-  return {
-    data: {
-      [address]: 9999,
-    },
-  };
-};
-
 export async function mockBtcBalanceQuote(
   mockServer: Mockttp,
   address: string = DEFAULT_BTC_ACCOUNT,
 ) {
   return await mockServer
-    .forGet(/https:\/\/api\.blockchair\.com\/bitcoin\/addresses\/balances/u)
-    .withQuery({
-      addresses: address,
+    .forPost(/https:\/\/.*\.btc.*\.quiknode\.pro/u)
```

Comment:
I think we should add the `^` and `$` achors.

Reply:
I've added a `^` since we could have some elements qfter the `quiknode.pro/` part.

This request is also constrained with another filter for the `bb_getaddress` method, so not all requests should be intercepted here